packages/ods/src/style/_drawer.scss

```diff
@@ -0,0 +1,130 @@
+$animation-duration: 0.3s;
+
+@mixin ods-drawer() {
+  all: unset;
+  display: flex;
+  position: fixed;
+  z-index: 1;
```

Comment:
we should start to define ODS z-indexes to ensure proper order between all our overlays